src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,69 +1,81 @@
-                    isOpen && isFocusVisible && FOCUS_STYLE,
-                ])}
-            >
-                <IconQuestion size={IconSize.Size16} />
-            </button>
-            {isOpen && (
-                <div
-                    {...tooltipAriaProps}
-                    data-test-id="tooltip"
-                    ref={setTooltipElement}
-                    style={styles.popper}
-                    {...attributes.popper}
-                    className="tw-p-4 tw-border tw-border-black-10 tw-bg-white tw-rounded-md tw-shadow-mid dark:tw-bg-black-90 dark:tw-text-white tw-z-20"
-                >
-                    {tooltip}
+            <div className="tw-inline-block tw-max-w-[200px] tw-bg-black-100 dark:tw-bg-white tw-rounded-md tw-shadow-mid tw-text-white dark:tw-text-black-100 tw-z-20">
+                {brightHeader && <BrightHeader variant={brightHeader} />}
+                <div data-test-id="tooltip" className="tw-p-4">
+                    {heading && headingIcon && (
+                        <h4 className="tw-flex tw-text-m tw-font-bold tw-mb-1">
+                            <span className="tw-mr-1.5">
+                                {headingIcon && cloneElement(headingIcon, { size: IconSize.Size20 })}
+                            </span>
+                            {heading}
+                        </h4>
+                    )}
+                    <div className="tw-flex">
+                        {tooltipIcon && (
+                            <span className="tw-flex-shrink-0 tw-mr-1">
+                                {cloneElement(tooltipIcon, { size: IconSize.Size16 })}
+                            </span>
+                        )}
+                        <p className="tw-text-s">{tooltip}</p>
+                    </div>
+                    {linkUrl && (
+                        <a
+                            href={linkUrl}
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            className="tw-text-xs tw-text-black-40 dark:tw-text-black-80 tw-underline tw-mt-1"
+                        >
+                            {linkLabel ?? "Click here to learn more."}
+                        </a>
+                    )}
+                    {showButtons && (
+                        <div className="tw-grid tw-grid-cols-2 tw-gap-x-1 tw-mt-4">
+                            <Button style={ButtonStyle.Secondary} size={ButtonSize.Small} onClick={buttons[0].action}>
+                                {buttons[0].label}
+                            </Button>
+                            <Button style={ButtonStyle.Primary} size={ButtonSize.Small} onClick={buttons[1].action}>
+                                {buttons[1].label}
+                            </Button>
```

Comment:
I see two cases that should be covered here:
- there's only one button defined which would be displayed _primary_
- there are >= 2 buttons, in which case the first would be displayed _primary_ and the second _secondary_

Reply:
On the mockups, the secondary button comes first and the primary one comes second. Did I understand that correctly?

Reply:
Yes, I just thought that from a developers perspective I'd put the button I wanted to be primary first in the array and the secondary one in second place. In the case where only one button is defined, I'd expect it to be primary and pass in either `[button]` or `[button, undefined]`.

Reply:
That makes sense. Simone also told me she doesn't expect a situation where there's only one button displayed. Should we be proactive and make it future-proof by allowing this specific use-case, though?